packaging/rpm/microshift.spec

```diff
@@ -46,6 +46,17 @@ BuildRequires: make
 BuildRequires: policycoreutils
 BuildRequires: systemd
 
+# DO NOT REMOVE
+#
+# Without the dependency on the compiler via this file, the production
+# build pipeline does not work. Sometimes that pipeline requires
+# compiling with a version of go that is not released into public RHEL
+# repos, yet, so we do not specify a version here. Instead, the
+# version is checked in the prep step below.
+#
+BuildRequires: golang
+# DO NOT REMOVE
```

Comment:
I think we can remove `DO NO REMOVE` statements. The comment seems to be enough on explaining why.